sql/table.cc

```diff
@@ -2473,6 +2473,9 @@ int TABLE_SHARE::init_from_binary_frm_image(THD *thd, bool write,
       field_data_type_info_array.parse(old_root, share->fields,
                                        extra2.field_data_type_info))
     goto err;
+  /*
```

Comment:
The comment sticks to the wrong block: it's fine to have no space before the for loop, but not fine to have a space between for and a comment block, and at the same time no empty line in between goto and a comment.

Either way to fix this is fine.